src/modelinstance.cpp

```diff
@@ -351,16 +351,47 @@ static Status applyLayoutConfiguration(const ModelConfig& config, std::shared_pt
     return StatusCode::OK;
 }
 
-const std::string RT_INFO_KEY{"model_info"};
+ov::AnyMap ModelInstance::getRTInfo(std::vector<std::string> path) {
```

Comment:
Why is this argument called path? Singular `path` along with vector type is misleading, but perhaps I'm missing something.